x-pack/plugin/esql/src/main/java/org/elasticsearch/xpack/esql/plugin/ComputeResponse.java

```diff
@@ -11,19 +11,56 @@
 import org.elasticsearch.common.io.stream.StreamInput;
 import org.elasticsearch.common.io.stream.StreamOutput;
 import org.elasticsearch.compute.operator.DriverProfile;
+import org.elasticsearch.core.TimeValue;
 import org.elasticsearch.transport.TransportResponse;
 
 import java.io.IOException;
 import java.util.List;
+import java.util.UUID;
 
 /**
  * The compute result of {@link DataNodeRequest} or {@link ClusterComputeRequest}
  */
 final class ComputeResponse extends TransportResponse {
     private final List<DriverProfile> profiles;
 
+    // for use with ClusterComputeRequests (cross-cluster searches)
+    private final TimeValue took;  // overall took time for a specific cluster in a cross-cluster search
+    public final int totalShards;
+    public final int successfulShards;
+    public final int skippedShards;
+    public final int failedShards;
+
+    public UUID uniqueId;
+
     ComputeResponse(List<DriverProfile> profiles) {
+        this(profiles, null, null, null, null, null);
+    }
+
+    @Override
+    public String toString() {
```

Comment:
Could you put this further down? Just not between the ctors.

Reply:
Yeah, sorry. Debugging left over. I'll remove it since I don't need it any more.